Samples/2a-arm-validation/shell/stderr.txt

```diff
@@ -26,6 +26,8 @@ ERROR (XmsExamplesProvidedValidation/M2022/SDKViolation): Please provide x-ms-ex
     - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-storage/2015-06-15/swagger/storage.json:362:8 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/regenerateKey"].post.operationId)
 ERROR (XmsExamplesProvidedValidation/M2022/SDKViolation): Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations. Operation: 'Usage_List'
     - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-storage/2015-06-15/swagger/storage.json:412:8 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Storage/usages"].get.operationId)
+WARNING (LROStatusCodesValidation/blah/SDKViolation): 200/201 Responses of long running operations must have a schema definition for return type. OperationId: 'StorageAccounts_Create', Response code: '200'
```

Comment:
i looked into this json file and I saw a ref for the operation https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json#L117, am I missing something? Not sure why the warning is reported.